examples/README.md

```diff
@@ -80,18 +80,19 @@ Example | File | Description
 
 ## 2D Rendering
 
-Example | File | Description
---- | --- | ---
-`contributors` | [`2d/contributors.rs`](./2d/contributors.rs) | Displays each contributor as a bouncy bevy-ball!
-`many_sprites` | [`2d/many_sprites.rs`](./2d/many_sprites.rs) | Displays many sprites in a grid arragement! Used for performance testing.
-`mesh2d` | [`2d/mesh2d.rs`](./2d/mesh2d.rs) | Renders a 2d mesh
-`mesh2d_manual` | [`2d/mesh2d_manual.rs`](./2d/mesh2d_manual.rs) | Renders a custom mesh "manually" with "mid-level" renderer apis.
-`rect` | [`2d/rect.rs`](./2d/rect.rs) | Renders a rectangle
-`sprite` | [`2d/sprite.rs`](./2d/sprite.rs) | Renders a sprite
-`sprite_sheet` | [`2d/sprite_sheet.rs`](./2d/sprite_sheet.rs) | Renders an animated sprite
-`text2d` | [`2d/text2d.rs`](./2d/text2d.rs) | Generates text in 2d
+Example | File                                               | Description
+--- |----------------------------------------------------| ---
+`contributors` | [`2d/contributors.rs`](./2d/contributors.rs)       | Displays each contributor as a bouncy bevy-ball!
+`many_sprites` | [`2d/many_sprites.rs`](./2d/many_sprites.rs)       | Displays many sprites in a grid arragement! Used for performance testing.
+`mesh2d` | [`2d/mesh2d.rs`](./2d/mesh2d.rs)                   | Renders a 2d mesh
+`mesh2d_manual` | [`2d/mesh2d_manual.rs`](./2d/mesh2d_manual.rs)     | Renders a custom mesh "manually" with "mid-level" renderer apis.
+`rect` | [`2d/rect.rs`](./2d/rect.rs)                       | Renders a rectangle
+`sprite` | [`2d/sprite.rs`](./2d/sprite.rs)                   | Renders a sprite
+`sprite_sheet` | [`2d/sprite_sheet.rs`](./2d/sprite_sheet.rs)       | Renders an animated sprite
+`text2d` | [`2d/text2d.rs`](./2d/text2d.rs)                   | Generates text in 2d
 `sprite_flipping` | [`2d/sprite_flipping.rs`](./2d/sprite_flipping.rs) | Renders a sprite flipped along an axis
-`texture_atlas` | [`2d/texture_atlas.rs`](./2d/texture_atlas.rs) | Generates a texture atlas (sprite sheet) from individual sprites
+`texture_atlas` | [`2d/texture_atlas.rs`](./2d/texture_atlas.rs)     | Generates a texture atlas (sprite sheet) from individual sprites
```

Comment:
could you remove those whitespace changes (and the extra `-`)?

Reply:
Done 👍🏻. Apparently my IDE tried to prettify the table 😅.